src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/tests/test_resource.py

```diff
@@ -260,3 +261,70 @@ def body(self):
             JMESPathCheck("length([?name=='Microsoft.Network/SkipPseudoVipGeneration'])", 1)
             ])
 
+class PolicyScenarioTest(ResourceGroupVCRTestBase):
+
+    def __init__(self, test_method):
+        super(PolicyScenarioTest, self).__init__(__file__, test_method)    
+        self.resource_group = 'azurecli-policy-test-group'
+       
+    def test_resource_policy(self):
+        self.execute()
+
+    def body(self):
+        policy_name = 'azurecli-test-policy'
+        policy_display_name = 'test_policy_123'
+        policy_description = 'test_policy_123' 
+        curr_dir = os.path.dirname(os.path.realpath(__file__))
+        rules_file = os.path.join(curr_dir, 'sample_policy_rule.json').replace('\\', '\\\\')
+        #create a policy
+        self.cmd('resource policy create -n {} --rules {} --display-name {} --description {}'.format(
+            policy_name, rules_file, policy_display_name, policy_description), checks=[
+                JMESPathCheck('name', policy_name),
+                JMESPathCheck('displayName', policy_display_name),
+                JMESPathCheck('description', policy_description)
+                ])
+        
+        #list and show it
+        self.cmd('resource policy list', checks=[
+            JMESPathCheck("length([?name=='{}'])".format(policy_name),1)
+            ])
+        self.cmd('resource policy show -n {}'.format(policy_name), checks=[
+            JMESPathCheck('name', policy_name),
+            JMESPathCheck('displayName', policy_display_name)
+            ])
+
+        #create a policy assignment on a resource group
+        policy_assignment_name = 'azurecli-test-policy-assignment'
+        policy_assignment_display_name = 'test_assignment_123'
+        self.cmd('resource policy assignment create --policy {} -n {} --display-name {} -g {}'.format(
+            policy_name, policy_assignment_name, policy_assignment_display_name, self.resource_group), checks=[
+                JMESPathCheck('name', policy_assignment_name),
+                JMESPathCheck('displayName', policy_assignment_display_name),
+                ])
+
+        # list at subscription won't work
```

Comment:
This is a bad comment. I have updated to `listing at subscription level won't find the assignment made at a resource group`